homework_weekend5/src/ba/bitcamp/homework_weekend5/task1/StoryCharacter.java

```diff
@@ -0,0 +1,86 @@
+package ba.bitcamp.homework_weekend5.task1;
+
+/**
+ * Describes a story character
+ * @author ajla
+ *
+ */
+public class StoryCharacter extends StoryElement {
+	private boolean sex;
+	private boolean isAlive;
+	private boolean isCapable;
+	/**
```

Comment:
Java doc should start with a sentence describing method/constructor.... Sentence must end with a `.`.